File: FIRE DRILL ONE/JAVASCRIPT/TaskSix.js
for(let count =1; count <=10; count++){
	if (count % 4 == 0){
	console.log(count)
	let fixCount = count
		for(let counter = 1; counter <= 5; counter++){
			let multiple = count * fixCount
			console.log(multiple + " ")
			count = multiple
		}
	}

}

File: FIRE DRILL ONE/JAVA/TaskSix.java
public class TaskSix{
	public static void main(String[] args){
		for(int count =1; count <=10; count++){
			if(count % 4 == 0){
			System.out.print(count);
				int fixCount = count;
				for(int counter = 1; counter <= 6; counter++){
					int multiple = count * fixCount;
						System.out.print(" ");
						System.out.print(multiple);
						count = multiple;
				}
			}
		}

	}


}